resources/assets/js/components/FriendsList.vue

```diff
@@ -129,13 +129,16 @@
                     text: 'This will delete @' + friend.username + ' from your list of friends',
                     type: 'warning',
                     showCancelButton: true
-                }, function () {
+                }, () => {
                     vm.$http.delete('/api/conferences/' + vm.conferenceId + '/' + vm.key + '/' + friend.id)
-                        .then(function () {
-                            vm.list.$remove(friend)
-                        });
+                        .then(() => { vm.list.$remove(friend) });
                 });
             },
+
+            metFriend: function (friend) {
+                this.$http.patch('/api/conferences/' + this.conferenceId + '/' + this.key + '/' + friend.id, { met: ! friend.met })
```

Comment:
This would indicate it's setting it to the inverse, which makes for a possibly incorrect method name. If we intend for it to set to the inverse, let's name it something different; if it's really just about marking a friend as met, let's make it explciit and have it be `met: true`